src/gennylib/include/gennylib/ExecutionStrategy.hpp

```diff
@@ -32,8 +31,9 @@ class ActorContext;
  * A small wrapper for running Mongo commands and recording metrics.
  *
  * This class is intended to make it painless and safe to run mongo commands that may throw
- * operation_exceptions. It maintains a timer for successful operations, an always-updated genny
- * gauge for failed operations, and an amortized guage for all operation. The ExecutionStrategy also
+ * boost exceptions.
+ *
+ * The ExecutionStrategy also
```

Comment:
[nit] Mind fixing the line ragging? My mom designed brochures for a living before she retired. :)